src/js/components/Diagram/stories/Progressing.js

```diff
@@ -72,13 +76,20 @@ export const Progressing = () => {
               ))}
             </Box>
           </Box>
-          <Diagram connections={connections} />
+          <Diagram
+            animation={{ type: 'draw', duration: 2000 }}
+            connections={connections}
```

Comment:
The connections are being drawn too fast, we should transit it slower.

Reply:
Progressing story now shows the animation transiting slower.